src/blindrsa.ts

```diff
@@ -188,6 +188,17 @@ export class BlindRSA {
     ): Promise<Uint8Array> {
         const algorithmName = privateKey.algorithm.name;
         privateKey.algorithm.name = BlindRSA.NATIVE_SUPPORT_NAME;
+        // only creates a copy if the private key algorithm name cannot be overwritten
+        if (privateKey.algorithm.name !== BlindRSA.NATIVE_SUPPORT_NAME) {
+            const privateKeyCopy = await crypto.subtle.exportKey('pkcs8', privateKey);
```

Comment:
Where does this key come from? It might fail if `privateKey.extractable` is false.

Reply:
it is passed by the caller. the error will propagate if the key is not extractable